.gitignore

```diff
@@ -11,3 +11,4 @@ radio/src/bootloader/bootloader_ramBoot*
 radio/src/lua/lua_fields_*.txt
 radio/src/lua/lua_exports_*.txt
 build*/
+/debian/
```

Comment:
http://stackoverflow.com/questions/24139478/when-to-use-leading-slash-in-gitignore

Summary: leading slash means specifically the mentioned resource (eg. only the `/debian/` folder found at the root of the repo).  No leading slash can include stuff matching the pattern in any subfolder (eg. any `*debian/` folder found in the root or any of the subfolders of the repo).